sdk/security_keyvault/src/keys/operations/get_key.rs

```diff
@@ -0,0 +1,32 @@
+use crate::prelude::*;
+
+operation! {
+    GetKey,
+    client: KeyClient,
+    ?version: String
+}
+
+impl GetKeyBuilder {
+    pub fn into_future(mut self) -> GetKey {
+        Box::pin(async move {
+            let mut uri = self.client.client.vault_url.clone();
+            let path = if let Some(ver) = self.version {
+                format!("keys/{}/{}", self.client.name, ver)
+            } else {
+                format!("keys/{}", self.client.name)
+            };
```

Comment:
FWIW, this doesn't hurt but it's not necessary. The Key Vault service already correctly handles if the version is elided and you try to GET something like `/keys/{key-name}//?api-version=7.1`.

Reply:
This was just refactored original code, but I will address.